src/index.js

```diff
@@ -110,9 +110,7 @@ class MediaQuery extends React.Component {
     const hasMergeProps = Object.keys(props).length > 0
     const childrenCount = React.Children.count(this.props.children)
     const wrapChildren = this.props.component ||
-      childrenCount > 1 ||
       typeof this.props.children === 'string' ||
```

Comment:
Do we need to wrap strings still? I thought react will wrap these in a span now.

Reply:
Good catch, you're right. Fixed this in latest commit.